.github/workflows/deploy-development.yaml

```diff
@@ -28,6 +49,9 @@ jobs:
       - name: Collect static assets 
         working-directory: ./src
         run: docker compose run app python manage.py collectstatic --no-input
+      - name: Run Django migrations 
```

Comment:
Fyi this is the wrong spot. You want to do this after the push below (name: Deploy to cloud.gov sandbox)

Reply:
Updated